src/System.IO.Abstractions.TestingHelpers/IMockFileDataAccessor.cs

```diff
@@ -8,6 +8,14 @@ namespace System.IO.Abstractions.TestingHelpers
     /// </summary>
     public interface IMockFileDataAccessor : IFileSystem
     {
+        /// <summary>
+        /// Adjust the times of the <paramref name="fileData"/>.
+        /// </summary>
+        /// <param name="fileData">The <see cref="MockFileData"/> for which the times should be adjusted.</param>
+        /// <param name="timeAdjustments">The adjustments to make on the <see cref="MockFileData"/>.</param>
+        /// <returns>The adjusted file.</returns>
+        MockFileData AdjustTimes(MockFileData fileData, TimeAdjustments timeAdjustments);
```

Comment:
The interface is a little confusing. The method in place modifies the given parameter `fileData` and also returns it.
At first when I read some usages of this method further in the changes, I was wondering why sometimes the return value is used and sometimes not. At least this confused me and I needed to check what was going on

Reply:
Is keeping the interface as is, OK for you, @siprbaum ?

Reply:
Yes, it is